packages/router/addon/index.js

```diff
@@ -1,9 +1,8 @@
 /*
-  This code is adapted from ember-engines/addon/-private/router-ext.js.
+  This cod- is adapted from ember-engines/addon/-private/router-ext.js.
 */
 import EmberRouter from '@ember/routing/router';
-import { registerWaiter, unregisterWaiter } from '@ember/test';
-import { DEBUG } from '@glimmer/env';
+import { waitForPromise } from '@ember/test-waiters';
```

Comment:
@rwjblue is it safe for us to use these new APIs?

Reply:
I believe this is a breaking change, requiring users to include `@ember/test-waiters`. We now include the `@ember/test-waiters` as an dep of this add-on. I believe that is the correct approach... let me know if it is not.

Reply:
> @rwjblue is it safe for us to use these new APIs?

It is safe if we add `@ember/test-waiters` to `@embroider/router`'s `dependencies`.